### DISCUSSION TITLE
Admin and User creation via Keycloak

### DISCUSSION THREAD

**Discussion by Zombexx** [drakkan/sftpgo#1607] [Q&A]
Hi.
I connected Keycloak and SFTPGO. User creation works like a charm.
Now i want to differentiate between User and Admin creation. 

I know that i have to create a event with an admin template. I just dont quite understand how i can differentiate between users and admins. 

My guess is that i could use a "role" but what exactly are the parameter? (e.g. role=user / role=admin)

So that i change

{
    "username": "{{Name}}",
    "email": "{{Email}}",
    "status": 1,
    
to 

{
    "username": "{{Name}}",
    "email": "{{Email}}",
    "role" : "{{Role}}",
    
  ?

**Comment by baranabi** [comment]
Unless changes have been made in the mean time, according to @drakkan 's reply [here](https://github.com/drakkan/sftpgo/issues/1179#issuecomment-1418808891), the dynamic management of users only supports normal users and not admins.

**Reply by Zombexx** [reply]
Thank you :)

**Comment by drakkan** [comment]
> Unless changes have been made in the mean time, according to @drakkan 's reply [here](https://github.com/drakkan/sftpgo/issues/1179#issuecomment-1418808891), the dynamic management of users only supports normal users and not admins.

My old answer is no longer accurate

**Comment by leotog** [comment]
so, can i set ldap auth plugin to WebAdmin?
